src/main/java/com/javabom/baseballgame/BaseBallApplication.java

```diff
@@ -0,0 +1,27 @@
+package com.javabom.baseballgame;
+
+import com.javabom.baseballgame.game.BaseBallGame;
+import com.javabom.baseballgame.game.PredicateResult;
+import com.javabom.baseballgame.number.ShuffledNumberGenerator;
+import com.javabom.baseballgame.view.InputView;
+import com.javabom.baseballgame.view.OutputView;
+
+public class BaseBallApplication {
+    public static void main(String[] args) {
+        BaseBallGame baseBallGame = new BaseBallGame(new ShuffledNumberGenerator());
+
+        OutputView.printAnswerBaseBallNumbers();
+
+        int tryNumber = 1;
+
+        while (true) {
+            PredicateResult predicateResult = baseBallGame.tryToPredicate(InputView.inputUserTryNumbers(), tryNumber++);
+            OutputView.printPredicateResult(predicateResult);
+
+            if (predicateResult.isAnswer()) {
+                OutputView.printLastTryNumber(predicateResult.getTryNumber());
+                break;
+            }
+        }
```

Comment:
```suggestion
        boolean isRemaining = true;
        while (isRemaining) {
            PredicateResult predicateResult = baseBallGame.tryToPredicate(InputView.inputUserTryNumbers(), tryNumber++);
            OutputView.printPredicateResult(predicateResult);
            isRemaining =  predicateResult.isAnswer();
            OutputView.printLastTryNumber(predicateResult.getTryNumber());
        }
```

이렇게하면 인덴트 규칙 지킬 수 있을것같아!